### PR TITLE
✨Add botinfo command

### PR DESCRIPTION
**Pull request by WilliamAcosta1**
# Description
Basic command to view the bot status

# Test

## Set of tests

| Use case | Result |
| :------- | :----- |
| zb-botinfo | image |

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have added the corresponding tranaslations at least in english language
- [ ] I checked my command works properly using prefix and slash commands

### CODE REVIEW

File: src/modules/info/commands/botstats.ts [hunk 1]
```diff
@@ -3,61 +3,61 @@ import { EmbedBuilder } from "discord.js";
 import { config } from "../../../config/index.js";
 import { cpus, totalmem, arch } from "os";
 
-export class Botinfo extends Command {
+export class Botstats extends Command {
```
Comment (fernandomema) [inline src/modules/info/commands/botstats.ts:6]: Utiliza UpperCamelCase por favor. BotStatus

File: src/modules/info/commands/botstats.ts [hunk 2]
```diff
@@ -3,61 +3,61 @@ import { EmbedBuilder } from "discord.js";
...
 
     categories = ['information'];
-    examples: string[] = ['']; 
-    aliases = ["botstatus"]; 
+    examples: string[] = [];
```
Comment (fernandomema) [inline src/modules/info/commands/botstats.ts:9]: No borres el ejemplo vacío. Este se utiliza para crear un ejemplo sin parámetros.

**Review by fernandomema — CHANGES_REQUESTED** [review]

File: src/modules/info/commands/botstats.ts (hunk 1 above)
Comment (WilliamAcosta1) [reply src/modules/info/commands/botstats.ts]: Alr

File: src/modules/info/commands/botstats.ts (hunk 2 above)
Comment (WilliamAcosta1) [reply src/modules/info/commands/botstats.ts]: Alr

**Review by fernandomema — APPROVED** [review]

**Pull request merged**